### PR TITLE
feat: add confirmation when deleting collections and objects

### PR DESCRIPTION
**Pull request by arnevandoorslaer**

### CODE REVIEW

File: packages/solid-crs-manage/lib/features/collection/collection-root.component.ts [hunk 1]
```diff
@@ -338,6 +371,25 @@ export class CollectionRootComponent extends RxLitElement {
...
+          align-items: center;
+          padding: 40px;
+          display: flex;
+          flex-direction: column;
+          justify-content: center;
+        }
+        #delete-popup div[slot="content"] div button{
+          width: 233px;
+          height: 45px;
+          margin: 40px 5px 0;
```
Comment (lem-onade) [inline packages/solid-crs-manage/lib/features/collection/collection-root.component.ts:385]:
```suggestion
          margin: var(--gap-large) var(--gap-tiny) 0;
```

File: packages/solid-crs-manage/lib/features/collection/collection-root.component.ts [hunk 2]
```diff
@@ -338,6 +371,25 @@ export class CollectionRootComponent extends RxLitElement {
...
+        }
+        button.accent {
+          color: white;
```
Comment (lem-onade) [inline packages/solid-crs-manage/lib/features/collection/collection-root.component.ts:388]:
```suggestion
          color: var(--colors-foreground-inverse);
```

File: packages/solid-crs-manage/lib/features/collection/collection-root.component.ts [hunk 3]
```diff
@@ -338,6 +371,25 @@ export class CollectionRootComponent extends RxLitElement {
...
+          background-color: white;
+          align-items: center;
+          padding: 40px;
+          display: flex;
+          flex-direction: column;
+          justify-content: center;
+        }
+        #delete-popup div[slot="content"] div button{
+          width: 233px;
+          height: 45px;
```
Comment (lem-onade) [inline packages/solid-crs-manage/lib/features/collection/collection-root.component.ts:384]: Check if needed

File: packages/solid-crs-manage/lib/features/collection/collection-root.component.ts [hunk 4]
```diff
@@ -338,6 +371,25 @@ export class CollectionRootComponent extends RxLitElement {
         .description {
           margin-top: var(--gap-tiny);
         }
+        #delete-popup div[slot="content"] {
+          background-color: white;
+          align-items: center;
+          padding: 40px;
```
Comment (lem-onade) [inline packages/solid-crs-manage/lib/features/collection/collection-root.component.ts:377]:
```suggestion
          padding: var(--gap-large);
```

File: packages/solid-crs-manage/lib/features/collection/collection-root.component.ts [hunk 5]
```diff
@@ -338,6 +371,25 @@ export class CollectionRootComponent extends RxLitElement {
         .description {
           margin-top: var(--gap-tiny);
         }
+        #delete-popup div[slot="content"] {
+          background-color: white;
```
Comment (lem-onade) [inline packages/solid-crs-manage/lib/features/collection/collection-root.component.ts:375]:
```suggestion
          background-color: var(--colors-foreground-inverse);
```

File: packages/solid-crs-manage/lib/features/object/object-root.component.ts [hunk 6]
```diff
@@ -530,6 +562,25 @@ export class ObjectRootComponent extends RxLitElement {
...
+          align-items: center;
+          padding: 40px;
+          display: flex;
+          flex-direction: column;
+          justify-content: center;
+        }
+        #delete-popup div[slot="content"] div button{
+          width: 233px;
+          height: 45px;
+          margin: 40px 5px 0;
```
Comment (lem-onade) [inline packages/solid-crs-manage/lib/features/object/object-root.component.ts:576]:
```suggestion
          margin: var(--gap-large) var(--gap-tiny) 0;
```

File: packages/solid-crs-manage/lib/features/object/object-root.component.ts [hunk 7]
```diff
@@ -530,6 +562,25 @@ export class ObjectRootComponent extends RxLitElement {
           max-width: var(--gap-normal);
           height: var(--gap-normal);
         }
+        #delete-popup div[slot="content"] {
+          background-color: white;
+          align-items: center;
+          padding: 40px;
```
Comment (lem-onade) [inline packages/solid-crs-manage/lib/features/object/object-root.component.ts:568]:
```suggestion
          padding: var(--gap-large);
```

File: packages/solid-crs-manage/lib/features/object/object-root.component.ts [hunk 8]
```diff
@@ -530,6 +562,25 @@ export class ObjectRootComponent extends RxLitElement {
           max-width: var(--gap-normal);
           height: var(--gap-normal);
         }
+        #delete-popup div[slot="content"] {
+          background-color: white;
```
Comment (lem-onade) [inline packages/solid-crs-manage/lib/features/object/object-root.component.ts:566]: See other color comments

File: packages/solid-crs-manage/lib/features/object/object-root.component.ts [hunk 9]
```diff
@@ -530,6 +562,25 @@ export class ObjectRootComponent extends RxLitElement {
...
+          flex-direction: column;
+          justify-content: center;
+        }
+        #delete-popup div[slot="content"] div button{
+          width: 233px;
+          height: 45px;
+          margin: 40px 5px 0;
+        }
+        button.accent {
+          color: white;
```
Comment (lem-onade) [inline packages/solid-crs-manage/lib/features/object/object-root.component.ts:579]: See other color comments

**Review by lem-onade — CHANGES_REQUESTED** [review]

**Review by BelgianNoise — CHANGES_REQUESTED** [review]
^ the above

File: packages/solid-crs-manage/lib/i8n/nl-NL.json [hunk 10]
```diff
@@ -124,6 +124,21 @@
...
+    {
+        "key": "nde.features.collections.root.delete.confirm",
+        "locale": "nl-NL",
+        "value": "Verwijderen"
+    },
+    {
+        "key": "nde.features.collections.root.delete.cancel",
+        "locale": "nl-NL",
+        "value": "Annuleren"
+    },
```
Comment (lem-onade) [inline packages/solid-crs-manage/lib/i8n/nl-NL.json:136]: These can probably go under the same key, under "common" at the bottom of the file. No need to change it in this PR, perhaps keep in mind when doing https://github.com/netwerk-digitaal-erfgoed/solid-crs/issues/336

**Review by lem-onade — APPROVED** [review]

File: packages/solid-crs-manage/lib/i8n/nl-NL.json (hunk 10 above)
Comment (lem-onade) [reply packages/solid-crs-manage/lib/i8n/nl-NL.json:136]: (same key as the object confirm and cancel)

**Review by BelgianNoise — APPROVED** [review]

**Pull request merged**